cloudinit/cmd/status.py

```diff
@@ -324,6 +328,35 @@ def _get_systemd_status() -> Optional[UXAppStatus]:
     return None
 
 
+def _get_error_or_running_from_systemd_with_retry(
+    existing_status, max_wait=5
```

Comment:
In the original bug (on OpenStack) the fix-case the commenter mentioned was a sleep of 30 seconds. Should this max_wait be longer for slower platforms so we don't reproduce this same failure case?